Xamarin.Essentials/Calendar/Calendar.android.cs

```diff
@@ -40,7 +46,7 @@ static async Task<IReadOnlyList<ICalendar>> PlatformGetCalendarsAsync()
                     IsReadOnly = IsCalendarReadOnly((CalendarAccess)cur.GetInt(calendarsProjection.IndexOf(CalendarContract.Calendars.InterfaceConsts.CalendarAccessLevel)))
                 });
             }
-
+            cur.Dispose();
```

Comment:
I would probably suggest taking advantage of `using` statements instead (won't need to call `Dispose`) at all

Reply:
Addressed in latest commit